pages/common/rails-db.md

```diff
@@ -0,0 +1,35 @@
+# rails db
+
+> Rails database actions.
+
+- Access database console:
+
+`rails db`
+
+- Create databases defined in current environment:
```

Comment:
Perhaps this will read better?
```suggestion
- Create the databases defined in the current environment:
```

Also, define "current environment"? Is this an environment variable, a configuration file, or something else?